### DISCUSSION TITLE
GitHub

### DISCUSSION THREAD

**Discussion by ak-0283** [community/community#128995] [Repositories]
I am facing a problem in GitHub where I have created a folder name project under that folder I have created a new folder project 1 now I wanna host that project 1 folder not that project folder. How to solve this problem 😞

**Comment by davevad93** [comment]
Hi @ak-0283 ,

I guess you're talking about deploying your project in this [repo](https://github.com/ak-0283/projects)  which contains a folder named `Folder1` with inside the files needed to deploy the project to `gh-pages`. If that's the case the solution is to put the contents' of `Folder1` in the root of your repo, here's what the structure should look like:

/projects
│
├── /assets
├── index.html
├──preview.png
└── README.md

Note that you can only deploy to GitHub Pages one project per repository, you can't have multiple `index.html` or assets files for different projects in the same repository. If you have other projects to deploy, you need to create a repo for each project. Remember that `index.html` must always be in the root of your repository, otherwise it will show a blank page.

**Comment by ak-0283** [comment]
okay

**Comment by ak-0283** [comment]
brother i am facing a new problem i.e i can't know how to solve this. when i am going to pages section it looks like this
![Screenshot 2024-06-19 104612](https://github.com/community/community/assets/143527276/ef2f16bc-ec38-4378-888d-7ae3570387fa)
hope you got my point.

**Reply by davevad93** [reply]
Hey @ak-0283 , I took a look at your [website](https://ak-0283.github.io/Projects/Project%201/) and it looks good, but I guess you want to use a custom domain? If that's the case you need to first own a domain and then verify it for `GitHub Pages`, take a look at the following from the docs:

1. [About custom domains and GitHub Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/about-custom-domains-and-github-pages)
2. [Managing a custom domain for your GitHub Pages site](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site)
3. [Verifying your custom domain for GitHub Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages)
4. [Troubleshooting custom domains and GitHub Pages](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/troubleshooting-custom-domains-and-github-pages)
